src/Utilities.Mongo/Interfaces/IMongoContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MongoDB.Driver;
+using System.Threading.Tasks;
+
+namespace MatrTech.Utilities.Mongo.Interfaces
+{
+    public interface IMongoContext
```

Comment:
I wonder if the summary of the interface is replaced by the implemantion class.

Reply:
Will verify

Reply:
Will use interface summary:
![image](https://user-images.githubusercontent.com/11668365/124430549-98d2eb00-dd6f-11eb-88e8-267e906bb147.png)

Unless overridden by implementation:
![image](https://user-images.githubusercontent.com/11668365/124430502-88227500-dd6f-11eb-92dd-06b157f4a1e3.png)